mopidy_musicbox_webclient/static/js/process_ws.js

```diff
@@ -229,7 +229,7 @@ function processArtistResults (resultArr) {
     resultsToTables(resultArr, ARTIST_TABLE, resultArr.uri)
     var artistname = getArtist(resultArr)
     $('#h_artistname, #artistpopupname').html(artistname)
-    getArtistImage(artistname, '#artistviewimage, #artistpopupimage', 'extralarge')
+    coverArt.getArtistImage(resultArr[0].uri, '#artistviewimage, #artistpopupimage', 'extralarge', mopidy)
```

Comment:
This was an oversight on my part to start with, but we'll probably end up doing a lookup on the track at this point anyway so that we can ask last.fm for the track info first, and then retrieve the artist image using the artist's MusicBrainz ID that it provides.